### ISSUE TITLE
Select color for calender entry queries

### ISSUE THREAD

**Issue by fmuell2** [marcusolsson/grafana-calendar-panel#23]
It would be great if one could select the color of the calender entries per query. Right now i have multiple queries defined and they are colored as green, yellow and blue. This seems to be default setting. But it would be nice to define a query as red ( e.g. for emergency changes or grey for 3rd party vendor changes...)

**Comment by marcusolsson** [comment]
This should be fairly easy to do with field options. Thanks for highlighting this! 🙏

**Comment by Gman-0815** [comment]
Hello,
can we still expect an implementation? It is currently not possible to color an entry differently or I just can't get it to work.

**Comment by fmuell2** [comment]
I dont think this is possible right now. But you can do the following: You could edit the constant legacyClassicColors here https://github.com/marcusolsson/grafana-calendar-panel/blob/main/src/CalendarPanel.tsx#L304

and set the colors you would like for your queries

**Comment by Gman-0815** [comment]
> @Gman-0815 Yes we use it as a maintenance calender. Just hit me with a mail, i have updated my profile

Thx  i have send you a mail
